pootle/apps/pootle_translationproject/views.py

```diff
@@ -266,18 +267,19 @@ class TPBrowseView(TPDirectoryMixin, TPBrowseBaseView):
 
     @cached_property
     def items(self):
+        dirs_with_vfolders = []
         if 'virtualfolder' in settings.INSTALLED_APPS:
-            from virtualfolder.helpers import vftis_for_child_dirs
             dirs_with_vfolders = set(
-                vftis_for_child_dirs(self.object).values_list(
-                    "directory__pk", flat=True))
-        else:
-            dirs_with_vfolders = []
+                split_pootle_path(path)[2].split("/")[0]
```

Comment:
Does this work ok for nested directories?

```python
In [1]: path = u'/gd/allffox/obsolete/mail/chrome/messenger/cloudfile/YouSendIt/fileExceeds2GB.dtd.po'

In [2]: split_pootle_path(path)[2].split("/")[0]
Out[2]: u'obsolete'
```

I was expecting the immediate parent directory name.